src/referee/referee/referee_client.py

```diff
@@ -0,0 +1,37 @@
+import socket
+import binascii
+from vision.proto.messages_robocup_ssl_wrapper_pb2 import SSL_WrapperPacket
+
+class Client:
+    
+    def __init__(self, ip:str, port:int):
+        """Client that connects and receives messages from ssl-vision"""
+        
+        self.ip = ip
+        self.port = port
+
+    def connect(self):
+        """Binds the client with ip and port and configure to UDP multicast."""
+        
+        self.sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM, socket.IPPROTO_UDP)
+        self.sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+        self.sock.setsockopt(socket.IPPROTO_IP, socket.IP_MULTICAST_TTL, 128)
+        self.sock.setsockopt(socket.IPPROTO_IP, socket.IP_MULTICAST_LOOP, 1)
+        self.sock.bind((self.ip, self.port))
+
+        host = socket.gethostbyname(socket.gethostname())
+        self.sock.setsockopt(socket.SOL_IP, socket.IP_MULTICAST_IF, socket.inet_aton(host))
+        self.sock.setsockopt(socket.SOL_IP, socket.IP_ADD_MEMBERSHIP, 
+                socket.inet_aton(self.ip) + socket.inet_aton(host))
+        
+    def receive(self):
+        """Receive package and decode."""
+
+        try:
+            data, _ = self.sock.recvfrom(2048)
```

Comment:
Why wait a bigger size message? dont think there is a 2x buffer size message begin sent